cmd/serve.go

```diff
@@ -250,7 +259,16 @@ func runServ(c *cli.Context) error {
 					user.Name, requestedMode, repoPath)
 			}
 		}
-	}
+	} else  {
+                // if public and read
```

Comment:
It's very obvious to tell that comment and code below don't have same indentation.